rust/platform/BUILD.bazel

```diff
@@ -21,6 +21,26 @@ constraint_value(
     constraint_setting = ":wasi_version",
 )
 
+constraint_setting(
```

Comment:
I would actually like to avoid adding constraint values like this into `rules_rust`. I've been seeing more activity on https://github.com/bazelbuild/platforms/issues/38 and want to use whatever pattern comes from this issue for constraints like this.